### PR TITLE
display mentions as a link

### PR DESCRIPTION
**Pull request by SamSamskies**
resolves https://github.com/stemstr/Client/issues/133

Screenshot 2023-07-01 at 4 07 46 PM

### CODE REVIEW

**Comment by SamSamskies** [comment]
hmmm mention link is clickable on local, but not on preview branch. need to investigate.

**Comment by SamSamskies** [comment]
mention link is fixed.

File: components/NoteContent/MentionLink.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <Anchor
+      component={Link}
+      href={`/user/${pubkey}`}
+      underline={false}
+      onClick={(e) => e.stopPropagation()}
+    >
+      {anchorText}
+    </Anchor>
+  ) : (
+    <>nostrUri</>
```
Comment (michaelhall923) [inline components/NoteContent/MentionLink.tsx:52]: think you meant <>{nostrUri}</> ?

Comment (SamSamskies) [reply components/NoteContent/MentionLink.tsx:52]: hahaha i was looking at your comment for like 2 minutes thinking that's exactly what i wrote 😂. doh.

**Pull request merged**